filters-share/extended_css_ELEMHIDE.txt

```diff
@@ -1,3 +1,9 @@
+ppss.kr###custom_html-3
+ppss.kr###genesis-content > div,span:not(:has(header)):not([itemprop])
+ppss.kr###ppss_wing_banner_left
+ppss.kr##.ad-video
```

Comment:
Some YouTube embeds are popped-up later and embeds have parent node with `ad-video` class.